Public/Src/Engine/Processes/Internal/BinaryPaths.cs

```diff
@@ -28,13 +23,16 @@ public BinaryPaths()
 
             VerifyFileExists(DllNameX86);
             VerifyFileExists(DllNameX64);
+
+            DllDirectoryX86 = Path.GetDirectoryName(DllNameX86);
+            DllDirectoryX64 = Path.GetDirectoryName(DllNameX64);
         }
 
         private static void VerifyFileExists(string fileName)
         {
             if (!File.Exists(fileName))

```

Comment:
Should this use FileUtilities.Exists?

Reply:
We typically use File.Exists for
- our own files,
- not needed for performance,
- don't care about following symlink.

---
In reply to: [303165327](https://github.com/microsoft/BuildXL/pull/589#discussion_r303165327) [](ancestors = 303165327)